domains/authorization/Authorization.API/API/ValueObjects/OrganizationId.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using EnergyOrigin.Domain.ValueObjects;
+
+namespace API.ValueObjects;
+
+public class OrganizationId : ValueObject
+{
+    public Guid Value { get; }
+
+    public OrganizationId(Guid value)
+    {
+        if (value == Guid.Empty)
+            throw new ArgumentException("Value cannot be an empty Guid.", nameof(value));
+
+        Value = value;
+    }
+
+    public static OrganizationId Create(Guid value)
+    {
+        return new OrganizationId(value);
+    }
+}
```

Comment:
Er det ikke blot en guid? Kan man så ikke lave et parent objekt, der hedder "Id" og lade OrganizationId arve fra den? eller går det imod nogle principper?

Reply:
It might be possible to create some base class for all Id-like value objects. But I think the _Create_ and _Empty_ methods must reside in the specific sub-class. I don't know how much we save b y doing so?